platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2958,6 +2966,47 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		074A7F0B2277BD67001A62D1 /* Insert Mapbox Access Token */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"$(TARGET_BUILD_DIR)/$(INFOPLIST_PATH)",
+			);
+			name = "Insert Mapbox Access Token";
+			outputFileListPaths = (
```

Comment:
Since this script changes `Info.plist`, should it be included here?

Reply:
Yeah, it appears that including this will allow the app's Info.plist value to be overridden with the build script 👍